_docs/instructor/course_management/plagiarism.md

```diff
@@ -212,5 +212,12 @@ belongs to the new username.
 
 ![](/images/instructor/lichen/others.png)
 
+Instructors are also able to toggle "Anonymous Mode" by clicking the
+button on the top right, which would hide student names and ID's and
```

Comment:
Might be better to phrase this as "which hides student names and ID's by replacing them with a hash to allow....."